android/app/src/main/kotlin/coop/polypoly/polypod/FeatureFragment.kt

```diff
@@ -88,6 +88,17 @@ open class FeatureFragment : Fragment() {
     private lateinit var foregroundResources: ForegroundResources
     private lateinit var featureContainer: FeatureContainer
 
+    private val errorDialog: AlertDialog by lazy {
+        AlertDialog.Builder(context)
+            .setPositiveButton(
+                context?.getString(R.string.button_acknowledge)
+            ) {
+                _, _ ->
```

Comment:
Nit: The parameters of an anonymous function usually go on the same line as the opening brace, as below, but not terribly important.

```
.setPositiveButton(
    context?.what.ever
) { _, _ ->
    close()
}
```

Reply:
They (ktlint) didn't let me :D